docs/aws-professional/index.md

```diff
@@ -379,9 +378,12 @@ allow you to create and manage the following storage services:
     message block (SMB) protocol. File storage is used in a similar manner to
     EFS in the AWS platform.
 
-#### Glacier and Azure Storage
 
-Azure Storage does not offer a direct equivalent to AWS' long-term archival
+
+
+ 
+#### Glacier and Azure Storage 
+[Azure Storage Standard Archive](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-storage-tiers) offers a direct equivalent to AWS' long-term archival
```

Comment:
You should always remove the `en-us` piece of the URL so that it resolves automatically to the user's browser locale.